corehq/apps/locations/views.py

```diff
@@ -250,19 +250,34 @@ class NewLocationView(BaseLocationView):
 
     @property
     def parent_pages(self):
-        return [{
+        selected = self.location._id or self.location.parent_id
```

Comment:
If you click "Locations" from the new location screen, jump to the parent's position in the tree.